randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -1,857 +1,882 @@
+
+> Door to Crateria Power Bomb Room; Heals? False
+  * Layers: default
+  * Normal Door to Crateria Power Bomb Room/Door to Landing Site
+  > Landing Site Save Station
+      Trivial
+
+> Door to Crateria Tube; Heals? False
+  * Layers: default
+  * Super Missile Door to Crateria Tube/Door to Landing Site
+  > Landing Site Save Station
+      Any of the following:
+          Hi-Jump Boots or Screw Attack or Space Jump or Speed Booster
+          All of the following:
+              Morph Ball
+              Morph Ball Bombs or Power Bombs
+
+> Event - Win Game; Heals? False
+  * Layers: default
+  * Event Defeat Mother Brain
+  > Landing Site Save Station
+      Trivial
+
+> Landing Site Save Station; Heals? True; Spawn Point
+  * Layers: default
+  > Door to Gauntlet Access
+      Any of the following:
+          Screw Attack and Wall Jumps (Advanced)
+          Infinite Bomb Jump (Intermediate) and IBJ
+          All of the following:
+              Morph Ball and Speed Booster
+              Morph Ball Bombs or Power Bombs
+          Morph Ball and Power Bombs and Wall Jumps (Advanced)
+          Speed Booster and Shinespark (Beginner)
+  > Door to Parlor
+      Trivial
+  > Door to Crateria Power Bomb Room
+      Any of the following:
+          Space Jump
+          Speed Booster and Shinespark (Beginner)
+          Infinite Bomb Jump (Intermediate) and IBJ
+  > Door to Crateria Tube
+      Any of the following:
+          Hi-Jump Boots or Screw Attack or Space Jump or Speed Booster
+          All of the following:
+              Morph Ball
+              Morph Ball Bombs or Power Bombs
+  > Event - Win Game
+      After Win Game
+
+----------------
+Crateria Tube
+Extra - map_name: 795D4
+Extra - asset_id: 1
+> Door to Landing Site; Heals? False; Spawn Point
+  * Layers: default
+  * Normal Door to Landing Site/Door to Crateria Tube
+  > Door to Crateria Kihunter Room
+      Trivial
+
+> Door to Crateria Kihunter Room; Heals? False
+  * Layers: default
+  * Normal Door to Crateria Kihunter Room/Door to Crateria Tube
+  > Door to Landing Site
+      Trivial
+
+----------------
+Crateria Kihunter Room
+Extra - map_name: 7948C
+Extra - asset_id: 2
+> Door to Crateria Tube; Heals? False; Spawn Point
+  * Layers: default
+  * Normal Door to Crateria Tube/Door to Crateria Kihunter Room
+  > Door to Moat
+      Trivial
+  > Door to Elevator to Red Brinstar
+      Trivial
+
+> Door to Moat; Heals? False
+  * Layers: default
+  * Power Bomb Door to Moat/Door to Crateria Kihunter Room
+
+> Door to Elevator to Red Brinstar; Heals? False
+  * Layers: default
+  * Power Bomb Door to Elevator to Red Brinstar/Door to Crateria Kihunter Room
+  > Door to Crateria Tube
+      Trivial
+
+----------------
+Moat
+Extra - map_name: 795FF
+Extra - asset_id: 3
+> Door to Crateria Kihunter Room; Heals? False; Spawn Point
+  * Layers: default
+  * Normal Door to Crateria Kihunter Room/Door to Moat
+  > Door to West Ocean
+      Any of the following:

```

Comment:
Isn't there an HBJ as well here? I swear I've also seen a strat with springball.

Reply:
Oh, yeah. I totally forgot about the HBJ since I always do a CWJ. I do not know of the strat for the CWJ. I'll get onto that once I update randovania